components/Utils/Footer/index.tsx

```diff
@@ -43,6 +43,7 @@ const Footer = () => {
             </div>
             <p className='monospace m-0'>
               <small>Made with &#10084;</small>
+              <small css={{ color: 'black' }}> by Jeremiah</small>
```

Comment:
Also whut

Reply:
It easter egg

Reply:
use `text-primary` if u want to use this

Reply:
Didn't work when I tried it